libs/media-player/src/lib/formatted-currenttime.tsx

```diff
@@ -0,0 +1,28 @@
+import { asTwoDigitString } from './as-two-digit-string';
+
+interface FormattedMediaTimeProps {
+    timeInSeconds: number;
+}
+
+export const FormattedMediaTime = ({ timeInSeconds }: FormattedMediaTimeProps): JSX.Element => {
+    const numberOfSecondsInHour = 3600;
+
+    const hours =
```

Comment:
We definitely need a time library- maybe as part of a larger math library. I'm almost certain this logic exists on the 
- frontend
- backend